Code/Framework/AzFramework/Platform/Linux/platform_nativeui_linux.cmake

```diff
@@ -18,13 +18,16 @@ if (${PAL_TRAIT_LINUX_WINDOW_MANAGER} STREQUAL "xcb")
     set(LY_FILES_CMAKE
         Platform/Common/Xcb/azframework_xcb_files.cmake
     )
+    include(Platform/Common/Xcb/generate_splash_screen_header.cmake)
     set(LY_BUILD_DEPENDENCIES
         PRIVATE
             3rdParty::X11::xcb
             3rdParty::X11::xcb_xkb
             3rdParty::X11::xcb_xfixes
             3rdParty::X11::xkbcommon
             3rdParty::X11::xkbcommon_X11
+            xcb_image
```

Comment:
The `3rdParty::X11::` alias is added by this [function](https://github.com/o3de/o3de/blob/development/cmake/3rdParty/FindX11.cmake). It  basically appends that alias to all of the X11 libraries defined by [cmake](https://cmake.org/cmake/help/latest/module/FindX11.html). 
You should be using `3rdParty::X11:xcb_image` since its defined as an import target, so I don't know whats going on in the Docker script.